### PR TITLE
Remove all references to mobile data

### PR DESCRIPTION
**Pull request by openjck**
We might have some mobile data to show in the future, but it might be a
while.

### CODE REVIEW

**Comment by openjck** [comment]
@spasovski Can you help me spot-check this? It's the biggest change we've made in a while.

File: src/components/views/styl/Header.styl [hunk 1]
```diff
@@ -30,96 +27,25 @@ h1 {
...
     li {
-        padding: $navgroup-v-spacing $navgroup-h-spacing;
+        margin: 0 15px;
 
-        &:first-child {
-            padding-top: 2 * ($navgroup-v-spacing);
-        }
         &:last-child {
-            padding-bottom: 2 * ($navgroup-v-spacing);
+            margin-right: 0;
```
Comment (spasovski) [inline src/components/views/styl/Header.styl]: I forget if ensemble will be translated. Will need RTL considerations if so.

**Comment by spasovski** [comment]
Code looks good. Clean + readable. r+

File: src/components/views/styl/Header.styl (hunk 1 above)
Comment (openjck) [reply src/components/views/styl/Header.styl]: Good catch! We have no plans to translate it now, but may do so in the future. I'll remember this if/when we get there. :smiley:

**Pull request merged**